### PR TITLE
Game 3

### PR DESCRIPTION
**Pull request by Sandeep-1-Kumar**
updated imperative tense handler for swahili algorithm

### CODE REVIEW

File: Back-End/slu-polyglot/src/main/java/team4/slupolyglot/model/EnglishSwahiliTranslation.java [hunk 1]
```diff
@@ -134,10 +150,90 @@ public String computeNegativeStatements(String person, String verb, String tense
...
+                break;
+            }
+        }
+        return containsSubstring;
+    }
+
+    public String imperativeTenseHandler(String root, String person){
+        if(person.equals(GENERAL_PRONOUNS[1])){
+            if(checkDiSyllables(root))
+            {
```
Comment (fulvius31) [inline Back-End/slu-polyglot/src/main/java/team4/slupolyglot/model/EnglishSwahiliTranslation.java:175]: The `{` should go at the end of the if statement 
can you double check also in all the other part of this commit?

File: Back-End/slu-polyglot/src/main/java/team4/slupolyglot/model/SyllableCounter.java [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 public class SyllableCounter {
     public static int countSyllables(String word) {
+        if(word.toLowerCase().equals("kuongea"))
+        {
+            return 3;
```
Comment (fulvius31) [inline Back-End/slu-polyglot/src/main/java/team4/slupolyglot/model/SyllableCounter.java:7]: we can beautify it afterwards. Btw, we should not use `magic numbers` so can you explain in a variable what is 3? thanks

**Review by fulvius31 — CHANGES_REQUESTED** [review]
Thanks for your work!

I wrote some comments. Other than that, we need to do refactoring in the next sprint.

Did all the tests passed?

File: Back-End/slu-polyglot/src/main/java/team4/slupolyglot/model/SyllableCounter.java (hunk 2 above)
Comment (Sandeep-1-Kumar) [reply Back-End/slu-polyglot/src/main/java/team4/slupolyglot/model/SyllableCounter.java:7]: I have removed that part of code and added a modified version of it. Please check in the next commit change. Thank you.

File: Back-End/slu-polyglot/src/test/java/team4/slupolyglot/SwahiliTests.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import static org.junit.Assert.assertEquals;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class SwahiliTests {
+    @Test
+    public void testSwahiliTranslation() throws IOException {
+    BufferedReader reader = new BufferedReader(new FileReader("C:\\Users\\sande\\Desktop\\PSD\\git_ultra_pro\\Polyglot\\Back-End\\slu-polyglot\\src\\test\\java\\team4\\slupolyglot\\resources\\testdist.tsv"));
```
Comment (fulvius31) [inline Back-End/slu-polyglot/src/test/java/team4/slupolyglot/SwahiliTests.java:17]: To import the tsv file, take inspiration from 
https://github.com/fulvius31/Polyglot/blob/develop/Back-End/slu-polyglot/src/test/java/team4/slupolyglot/TranslationTests.java

the file has to be copied on https://github.com/fulvius31/Polyglot/tree/develop/Back-End/slu-polyglot/src/main/resources folder

**Review by fulvius31 — CHANGES_REQUESTED** [review]
Thanks for the code provided!

Just a small change on the test file so the test can run in every machine

**Comment by Sandeep-1-Kumar** [comment]
> Thanks for the code provided!
> 
> Just a small change on the test file so the test can run in every machine

Yes thanks for letting me know brother i have updated and pushed.

**Pull request merged**